File: python/meteroid/models/token_introspection_response.py
# this file is @generated
import dataclasses
import typing as t

from ..serialization import BaseModel


@dataclasses.dataclass
class TokenIntrospectionResponse(BaseModel):
    """Token introspection response as per RFC 7662"""

    active: bool

    client_id: t.Optional[str] = None

    exp: t.Optional[int] = None

    iat: t.Optional[int] = None

    scope: t.Optional[str] = None

    sub: t.Optional[str] = None

    token_type: t.Optional[str] = None

File: go/token_introspection_response.go
// this file is @generated
package meteroid

// Token introspection response as per RFC 7662
type TokenIntrospectionResponse struct {
	Active bool `json:"active"`

	ClientId *string `json:"client_id,omitempty"`

	Exp *int64 `json:"exp,omitempty"`

	Iat *int64 `json:"iat,omitempty"`

	Scope *string `json:"scope,omitempty"`

	Sub *string `json:"sub,omitempty"`

	TokenType *string `json:"token_type,omitempty"`
}
